### ISSUE TITLE
Workstation throwing bundler errors

### ISSUE THREAD

**Issue by tas50** [chef/chef-workstation#492]
0.9.20:

```
/opt/chef-workstation/bin/chef -v
Traceback (most recent call last):
	12: from /opt/chef-workstation/bin/chef:86:in `<main>'
	11: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/core_ext/kernel_gem.rb:67:in `gem'
	10: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/core_ext/kernel_gem.rb:67:in `synchronize'
	 9: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/core_ext/kernel_gem.rb:68:in `block in gem'
	 8: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/specification.rb:1420:in `activate'
	 7: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/specification.rb:1438:in `activate_dependencies'
	 6: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/specification.rb:1438:in `each'
	 5: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/specification.rb:1452:in `block in activate_dependencies'
	 4: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/specification.rb:1420:in `activate'
	 3: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/specification.rb:1438:in `activate_dependencies'
	 2: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/specification.rb:1438:in `each'
	 1: from /opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/specification.rb:1449:in `block in activate_dependencies'
/opt/chef-workstation/embedded/lib/ruby/2.6.0/rubygems/dependency.rb:313:in `to_specs': Could not find 'bundler' (2.0.2) required by your /Users/tsmith/dev/work/chef-workstation/Gemfile.lock. (Gem::MissingSpecVersionError)```

**Comment by tyler-ball** [comment]
Lets check if this is still an issue in the latest build

**Comment by tas50** [comment]
Looks like this is fixed in 0.9.22
